src/FeaturedComponents.jsx

```diff
@@ -51,6 +51,12 @@ return (
     </Header>
 
     <Items>
+      <Item>
+        <Widget
+            src="${REPL_ACCOUNT}/widget/ComponentCard"
```

Comment:
Hi @charleslavon, 
Based on the replacement configuration, what I've observed is that the `ComponentCard` component must exist on the `near` account. Similar to the other items on the list of featured components, it would render the same component for each item. This means that all of them should use the same `REPL_ACCOUNT` account.

Reply:
Hi @charleslavon, 
I'm following up on my previous message. Is there any additional feedback we should consider?
Thanks

Reply:
I see. I missed that you've passed in the specific meta pool component as a prop.

Reply:
@leomanza  Do you have a version of this component on testnet?

Reply:
@charleslavon  We dont have a testnet version but we have the production version that i can share with you https://near.org/meta-pool-official.near/widget/MetaPoolStakeEth. 

Is there anything else that you need so we can move forward with this?

Reply:
@sergiogaguero Yes, this repo is deployed into testnet and mainnet account as specified in the replacements.*.json files. Please add a new REPL tag in the `mainnet` `testnet` and `dev` [replacements files](https://github.com/near/near-discovery-components/blob/develop/replacements.mainnet.json).  You can then update the props to reference `{REPL_METAPOOL}/widget/MetaPoolStakeEth`

Reply:
Hi, @charleslavon 
If I understand correctly what is required on the latest message, once the MetaPoolStakeEth component has been created on both the testnet and mainnet, and properly referenced via the replacements.*.json file, it would allow users to access this component within the `Featured Components`, not only on near.org but also on test.near.org.
Is that correct?

Reply:
@charleslavon requested changes pushed
Please, let us know if changes are correct.